bip-0009/assignments.mediawiki

```diff
@@ -0,0 +1,37 @@
+==Deployments==
+
+List of proposed deployments.
+
+State can be defined, active, failed. Dates are in UTC.
+
+{| class="wikitable"
```

Comment:
I'm not suggesting to merely sort it, but to format it in a way that it is clear which bits are used at what times.